ui/src/components/schemas/SchemasTable.tsx

```diff
@@ -66,9 +72,10 @@ export function MySchemas() {
               Details
             </Link>
             <Link
-              to={generatePath(ROUTES.issueCredential.path, {
-                schemaID,
-              })}
+              to={{
+                pathname: generatePath(ROUTES.issueCredential.path),
+                search: `?${SCHEMA_SEARCH_PARAM}=${schemaID}`,
```

Comment:
[Ditto](https://github.com/0xPolygonID/sh-id-platform/pull/381/files#r1184865106).

Reply:
Check `SchemasTable`, line 77.

Reply:
What it's happening there?

Reply:
![Screenshot from 2023-05-04 14-13-16](https://user-images.githubusercontent.com/2691584/236200567-ce3ba691-c1ce-4371-932a-729145d45381.png)

Reply:
Nevermind, it's fixed now.